pkg/controller/restore_controller.go

```diff
@@ -520,6 +521,16 @@ func (r *restoreReconciler) runValidatedRestore(restore *api.Restore, info backu
 		return errors.Wrap(err, "fail to fetch CSI VolumeSnapshots metadata")
 	}
 
+	backupVolumeInfoMap := make(map[string]volume.VolumeInfo)
+	volumeInfos, err := backupStore.GetBackupVolumeInfos(restore.Spec.BackupName)
+	if err != nil || volumeInfos == nil {
+		restoreLog.WithError(err).Infof("Backup %s doesn't have volumeinfos metadata file.", restore.Spec.BackupName)
```

Comment:
It may not be an error b/c old backups do not have such  metadata

Reply:
Modified.